src/components/BasicLed.vue

```diff
@@ -369,11 +404,37 @@ export default {
 
     changes(cmjson) {
       let cm = null == cmjson ? {} : JSON.parse(cmjson)
-      return Object.keys(cm)
+      let chs = Object.keys(cm)
         .filter(k=>'audit'!=k)
         .reduce((a,c)=>
                 (a.push({field:c,old:cm[c][0],new:cm[c][1]}),a),[])
-    }
+
+      chs = chs.filter(v => {
+        let field
+        // if there are no changes - don't show the field
+        if(!v.old && !v.new) { 
+          return 0
+        }
+        if(field = this.getInfoFieldByName(v.field)) {
+          v.field = field.title
```

Comment:
this is breaking change
instead of changing the value of the `field` property, add a new property `title`, with the user visible name of the field

Reply:
Done